Sources/Fluent/SQLite/SQLiteSerializer.swift

```diff
@@ -1,9 +1,9 @@
 import SQLite
 
 /// SQLite-specific overrides for the GeneralSQLSerializer
-public class SQLiteSerializer: GeneralSQLSerializer {
+public class SQLiteSerializer<E: Entity>: GeneralSQLSerializer<E> {
     /// Serializes a SQLite data type.
-    public override func sql(_ type: Schema.Field.DataType, primaryKey: Bool) -> String {
+    public override func type(_ type: Field.DataType, primaryKey: Bool) -> String {
```

Comment:
if it serializes a sqlite data type, why not call this function `serialize`? `type` looks a little odd here